### PR TITLE
ci: Updater fixes for release branches

### PR DESCRIPTION
**Pull request by dansiegel**
GitHub Issue (If applicable): closes #

- n/a

## PR Type

What kind of change does this PR introduce?

- Project automation

## What is the current behavior?

Uno Sdk Updater is really tooled to run for dev only
Uno Sdk Updater runs once per hour

## What is the new behavior?

Uno Sdk Updater now uses a matrix to potentially create one or more PRs for dev and stable releases
Uno Sdk Updater now runs once every 6 hours. This will reduce the load on CI agents as some update PRs have triggered new builds due to a 2nd update just before the initial set of CI validations finished.

### CODE REVIEW

File: .github/workflows/uno-updater.yml [hunk 1]
```diff
@@ -20,12 +19,29 @@ on:
 jobs:
   manifest-update:
     runs-on: windows-latest
+    strategy:
+      matrix:
+        branch:
+          - main
+          - release/stable/5.3
```
Comment (jeromelaban) [inline .github/workflows/uno-updater.yml:26]: Let's use that instead, I thought it was used by triggers, but it's not.

```suggestion
          - release/stable/*
```

Comment (jeromelaban) [reply .github/workflows/uno-updater.yml:26]: Scratch that, wrong line.

File: .github/workflows/uno-updater.yml [hunk 2]
```diff
@@ -2,11 +2,10 @@ name: Update Uno Sdk
 
 on:
   schedule:
-    - cron: '0 * * * *'
+    - cron: '0 */6 * * *'
   pull_request:
     branches:
       - main
-      - release/stable/5.3
```
Comment (jeromelaban) [inline .github/workflows/uno-updater.yml:9]: I think we should still be executing it for release branches, with `release/stable/*`.

Comment (dansiegel) [reply .github/workflows/uno-updater.yml:9]: Just because we have a backport to a release branch doesn't mean the SDK updater needs to get triggered.

**Review by jeromelaban — APPROVED** [review]

**Comment by jeromelaban** [comment]
@mergifyio backport release/stable/5.3

**Pull request merged**